boards/arm/96b_carbon_nrf51/96b_carbon_nrf51-pinctrl.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 29)>,
+				<NRF_PSEL(UART_RX, 0, 11)>;
```

Comment:
Did we ever settle on the above style versus this one?

```
			psels = <NRF_PSEL(UART_TX, 0, 29)
				 NRF_PSEL(UART_RX, 0, 11)>;
```

I seem to recall @carlescufi had an opinion on this, but I can't remember what it was.

Reply:
I think nothing was agreed, but Linux seems to use

```
			psels = <NRF_PSEL(UART_TX, 0, 29)>,
				<NRF_PSEL(UART_RX, 0, 11)>;
```

Reply:
Just to be clear since I am going on vacation, I don't care personally. I just wanted to bring it up.